src/render/state.js

```diff
@@ -10,19 +10,17 @@ const {
  *
  * @param      {Function|Object}  newState  An object containing new keys/values
  *                                          or a function that will provide the same.
- * @param      {Function}         cb        A callback that may be provided to setState.
```

Comment:
Yeah, but React  doesn't call that callback during server side rendering

```jsx
      var React = require('react');
      var renderToString = require('react-dom/server').renderToString;

      var Component = React.createClass({
        componentWillMount: function() {
          console.log('set x to bar');
          this.setState({x: 'bar'}, function() {
            console.log('set x to foo');
            this.setState({x: 'foo'});
          });
        },

        render: function() {
          return <div>{`x is ${this.state.x}`}</div>;
        }
      });

      console.log(renderToString(<Component />));
```

console output
```
set x to bar
<div data-reactroot="" data-reactid="1" data-react-checksum="-8842717">x is bar</div>
```